models/yolo.py

```diff
@@ -232,6 +232,13 @@ def autoshape(self):  # add AutoShape module
     def info(self, verbose=False, img_size=640):  # print model information
         model_info(self, verbose, img_size)
 
+    def _apply(self, fn):
+        self = super()._apply(fn)
+        m = self.model[-1]  # Detect()
```

Comment:
Custom models may have a different module as the last one. Maybe a check needs to be added
```suggestion
        m = self.model[-1]  # Detect()
        if isinstance(m, Detect):
```